packages/connect-thegraph/subgraph/package.json

```diff
@@ -17,10 +17,11 @@
     "deploy-staging": "env STAGING=true scripts/deploy.sh aragon mainnet",
     "deploy-rinkeby": "scripts/deploy.sh aragon rinkeby",
     "deploy-rinkeby-staging": "env STAGING=true scripts/deploy.sh aragon rinkeby",
-    "deploy-ropsten": "scripts/deploy.sh aragon ropsten"
+    "deploy-ropsten": "scripts/deploy.sh aragon ropsten",
+    "deploy-xdai": "scripts/deploy.sh 1hive xdai"
   },
   "devDependencies": {
-    "@graphprotocol/graph-cli": "^0.18.0",
+    "@graphprotocol/graph-cli": "https://github.com/graphprotocol/graph-cli",
```

Comment:
Using the master branch has a feature to get stack traces on the graph dashboard logs.

Reply:
I'll check if they already release this item and update to that version if that's the case.

Reply:
They already released 0.19.0 🙌